Packages/vcs/vcs/utils.py

```diff
@@ -312,42 +312,6 @@ def _scriptrun(script, canvas=None):
                 vcs.process_src_element(s)
                 processing = False
     f.close()
-    # Ok now we need to double check the isolines
```

Comment:
@doutriaux1 I did not really understand that code, and something bad was happening there as far as I can remember. It made all tests pass. I will take a closer look.